### DISCUSSION TITLE
Align TextBox Baseline

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#11163] [Q&A] [answered]
Considering following code snippet:

```xaml
  <TextBlock FontSize="24">
    <TextBlock.Inlines>
      <Run Text="📐 H"/>
      <TextBox Text="Hello"
               FontSize="24"
                   />
      <Run Text="("
           FontSize="14"/>
      <Run Text="xxxx"
           FontSize="14"
           FontWeight="Bold"/>
      <Run Text=")"
           FontSize="14"/>
      <Run Text="H"/>
    </TextBlock.Inlines>
  </TextBlock>
```
will produce:

![immagine](https://user-images.githubusercontent.com/12531229/234917265-55e7753f-89aa-4f5c-8a68-97c6f33f163d.png)

How can I align text of TextBox to baseline of TextBlock?

Avalonia Version: 82c7c0d17b9f9cc633053cbf6bd0acddf28c8bab
Platform:  **Window 10**

**Comment by Gillibald** [comment]
If you are working with a fixed font and size you can set the TextBlock.BaselineOffset

**Reply by Gillibald** [reply, marked as answer]
You can also try to set each Inline's BaselineAlignment to Center

**Reply by workgroupengineering** [reply]
Thanks, it works for me
